documenting app templates (#151)
Basis-Theory/basistheory-integration-guides: collections/concepts/what-are-applications.md

Your Application’s API Key is used to authenticate your systems to the Basis Theory platform. The API Key can be used to make authenticated requests to our API directly, using Hosted Elements, or using the Basis Theory SDK. These API Keys follow a format similar to `key_4qUtg83907AnDem90aJSzcN`.

## Application Templates

To get you started quickly, we provide a set of Application Templates that cover the most common use cases. Once you click on
"Create Application", a modal will appear to let you select an Application Template depending on your needs. If you 

> have a particular use case or want to create an Application step by step, you can click the `Create Your Own` option. It

Reviewer: ```suggestion
have a particular use case or want to create an Application from scratch, you can click the `Create Your Own` option. It
```

is important to note that these templates will only pre-fill the fields in the Create Application page, and once created,
your Application has no relationship with the template in any way.

<img alt="Application Templates" src="/assets/images/concepts/application-templates.png">

> In this modal you can see we provide templates for the most common usages such as `Payments`, `Banking` and `PII Data`.
> However, we also have the `Starters` section which allows you to create your own Application step by step if you have 
> a custom use case or want to get started quickly with a `Full Access` Application. The last section is `Management`, which 
> provides a `Full Management Access` template to get all the managing permissions available.

Reviewer: IMO this is way too verbose and we don't need to describe every detail of the UI.

```suggestion
This modal displays several templates to quickly achieve common use cases, such as `Payments`, `Banking`, and `PII Data`. The `Full Access` template can be useful for creating a test Application with every permission, but it should not be used for production Applications. 
```

> provides a `Full Management Access` template to quickly get all the managing permissions available.

Author: cuz APs let you get started so fast it is blink and you miss it

The `Payments`, `Banking` and `PII` sections all provide four templates each with specific 
[Access Rules](https://developers.basistheory.com/concepts/what-are-access-rules/) for its objective. These four templates
are `Collect (PCI/Bank/PII) Data`, which enables the application to only create or update tokens in the given
[container](https://developers.basistheory.com/concepts/what-are-containers/); `Display (PCI/Bank/PII) Data`, which allows

> to read the sensitive raw data stored in tokens; `Use (PCI/Bank/PII) Data`, for applications that need to use tokens 

Reviewer 2: ```suggestion
applications to read the sensitive raw data stored in tokens; `Use (PCI/Bank/PII) Data`, for applications that need to use tokens 
```

> in a Reactor or Proxy and `Full (PCI/Bank/PII) Data Access`, which gives access to all token operations (create, update, read, use, delete)

Reviewer 2: ```suggestion
in a Reactor or Proxy; and `Full (PCI/Bank/PII) Data Access`, which gives access to all token operations (create, update, read, use, delete)
```

> within that container.

Reviewer 2: sorry, I had to look the proper use of `;` https://www.niu.edu/writingtutorial/punctuation/semicolon.shtml#:~:text=with%20independent%20clauses-,Use%20a%20semicolon%20to%20join%20two%20related%20independent%20clauses%20in,clear%20without%20the%20coordinating%20conjunction.
looks like we're using it wrong since each of these isn't a separate idea. how about we change this to a numbered list and end the first sentence for this with a `:`. like this:
These four templates are:
1. `Collect (PCI/Bank/PII) Data`, which enables the Application to only create or update tokens in the given
[container](https://developers.basistheory.com/concepts/what-are-containers/)
2. ...

Author: Use a semicolon to join two related independent clauses in place of a comma and a coordinating conjunction (and, but, or, nor, for, so, yet).

Author: Semicolons join ideas that are related and equal in grammatical structure.

Author: A semicolon is used in place of a comma to separate phrases or items in a list or series when the phrases or items themselves contain commas or are especially long

Author: i don't really agree, our use case is exactly what they are for

Reviewer 2: if `;` separate complete ideas with the same grammatical structure, I see two things wrong with how it is now:
1) this first sentence is not a complete idea `These four templates
are `Collect (PCI/Bank/PII) Data`, which enables the Application to only create or update tokens in the given
[container](https://developers.basistheory.com/concepts/what-are-containers/)` since it's missing the other 3 templates. you can't stop there or put that in a sentence and call it a complete idea

2) the final idea `and `Full (PCI/Bank/PII) Data Access`, which gives access to all token operations (create, update, read, use, delete)` starts with `and` which is pointed out as incorrect in the very first example of that site

Reviewer 2: lol you know what? I don't know anymore. it looks like it depends on who you ask. if you really want to use them, it's fine with me. but this confusion is exactly why I try to avoid them

Reviewer: Honestly, I would delete this entire paragraph. I feel describing the UI to this level of detail is unnecessary and most of this is done by the image we're embedding on the page. Keep it short and to the point, or readers will lose interest and skip the important parts.

> Remember these templates are more of recommendations that enable you to quickly get started, but you can update the fields

Reviewer 2: ```suggestion
Remember these templates are more of recommendations that enable you to get started quickly, but you can update the fields
```

> Remember these templates are more of recommendations that enable you to get started quickly, but you can update the fields
> that are pre-filled after selecting them as you like, and if none work for you, you can still create an Application
> that conforms to your specific needs. 

Reviewer: This needs some word-smithing:
```suggestion
Remember these templates only serve as recommendations that enable you to get started quickly, and you can customize any of the settings or rules. If none of these templates work for you, you can still create a custom Application to meet your specific requirements. 
```
